### DISCUSSION TITLE
一些有趣的AI網站

### DISCUSSION THREAD

**Discussion by alvin1226** [1101-GS3073-C/Python-Community-GS3073-C#645] [科技新知]
https://www.jasper.ai/art 這大家應該或多或少都聽過，可根據你提供的主題等等來叫AI幫你繪圖
https://www.synthesia.io/ 這個我覺得蠻酷的，可以用虛擬人像在你的影片中翻譯語言並說出來，感覺蠻有用的
https://repurpose.io/ 這個可以幫你把你在其他社群軟體所發過的內容加以修飾，並在其他社群軟體統一發出，對商家來說可以節省很多時間

**Comment by bww0401** [comment]
謝謝分享，感覺AI能做的事情真的越來越多，帶來很多便利

**Comment by liangyinglly** [comment]
謝謝分享 AI的運用真的很多元ㄟ
